mappings/net/minecraft/client/render/debug/GoalSelectorDebugRenderer.mapping

```diff
@@ -1,7 +1,16 @@
 CLASS net/minecraft/class_4205 net/minecraft/client/render/debug/GoalSelectorDebugRenderer
 	CLASS class_4206 GoalSelector
 		FIELD field_18782 pos Lnet/minecraft/class_2338;
-		FIELD field_18784 name Ljava/lang/String;
+		FIELD field_18784 label Ljava/lang/String;
+		FIELD field_18785 reachable Z
```

Comment:
What makes you think this is "reachable"? If it's true, the name is rendered in green, if false, gray. Maybe name it "active". I have no idea where you're getting "reachable" from.

Reply:
`reachable` is a reasonable thing to have for a goal equivalent to "achievable". How can a goal be enabled or disabled?